scripts/libs/utils.js

```diff
@@ -79,6 +79,33 @@ function getTestnetChains(chains = []) {
     }));
 }
 
+/**
+ * Get chains config for testnet.
+ * @param {*} chains - The list of chains to get the chain objects for. If this is empty, the default chains will be used.
+ * @returns {Chain[]} - The chain objects.
+ */
+function getMainnetChains(chains = []) {
```

Comment:
code duplication. reuse the same function, just make the path derivation dependent on env